includes/shortcodes.php

```diff
@@ -75,37 +75,37 @@ function shortcode_mdjm( $atts )	{
  *
  * Displays the Client Zone home page which will render event details if the client only has a single event
  * or a list of events if they have multiple events in the system.
- * 
+ *
  * @since	1.3
  *
  * @return	string
  */
 function mdjm_shortcode_home( $atts )	{
-	
+
 	if ( is_user_logged_in() ) {
 
 		global $mdjm_event;
-		
+
 		$mdjm_event = '';
-		
+
 		ob_start();
-		
+
 		$output = '';
-		
+
 		$client_id = get_current_user_id();
-		
+
 		mdjm_add_content_tag(
 			'event_action_buttons',
 			sprintf( __( '%s action buttons within %s', 'mobile-dj-manager' ), mdjm_get_label_singular(), mdjm_get_application_name() ),
 			'mdjm_do_action_buttons'
 		);
-		
+
 		if( isset( $_GET['event_id'] ) )	{
 			$mdjm_event = mdjm_get_event( $_GET['event_id'] );
-			
-			if( ! empty( $mdjm_event->ID ) )	{
+
+			if( ! empty( $mdjm_event->ID ) && intval($mdjm_event->client) === $client_id )	{
```

Comment:
This looks good however could you change the check so that it uses `absint` rather than `intval` as the client id should always be a positive integer

Reply:
Fixed conversion :)